### DISCUSSION TITLE
Processing large HTML Contents to PDF

### DISCUSSION THREAD

**Discussion by hmennen90** [spatie/browsershot#640] [Q&A]
I've to work with some very large Generated HTML Contents (~10MB).

When I try to process these, this Error Output shows up in the log.

Any Idea to solve it?

```

Output:
================


Error Output:
================
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
TimeoutError: Navigation timeout of 30000 ms exceeded
```

**Comment by grebe-vladin** [comment]
Have you solved this?

**Reply by hmennen90** [reply]
Splittet the HTML String using mb_str_split before processing it with Browsershot

**Reply by Waghabond** [reply]
You should be able to simply increase your webserver's timeout
